kimchi/src/circuits/polynomials/range_check/circuitgates.rs

```diff
@@ -1,11 +1,10 @@
 ///```text
-/// Range check field element structure:
+/// Range check circuit gates:
 ///
-///    Each field element a should be decomposed into three 88-bit limbs a0, a1, a2 s.t. a = a0a1a2 in
-///    little-endian byte order (i.e. a = a2*2^{2b} + a1*2^b + a0).
+///    The rang check gate is comprised of three circuit gates (RangeCheck0, RangeCheck1
```

Comment:
"rang"

also, why reintroduce RangeCheck2?

Reply:
Just temporarily- I will remove it properly and "atomically" all with one commit.